posts/posts-router.js

```diff
@@ -0,0 +1,94 @@
+const express = require("express");
+const db = require("../data/db.js");
+const router = express.Router();
+
+router.post("/", (req, res) => {
```

Comment:
great job with all the router logic and using the correct http codes for the relevant  successes and errors of the logic.